### PR TITLE
docs: fix typos and styling in lessons 7-14

### PR DESCRIPTION
**Pull request by bampakoa**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/tutorial/first-app/first-app-lesson-09.md [hunk 1]
```diff
@@ -71,14 +71,14 @@ For now, your app's new service uses the data that has, so far, been created loc
 In the **Edit** pane of your IDE:
 
 1.  In `src/app/home/home.component.ts`, from `HomeComponent`, copy the `housingLocationList` variable and its array value.
-1.  In `src/app/housing.service.ts`:
-    1.  Inside the `HousingService` class, paste the variable that you copied from `HomeComponent` in the previous step.
-    1.  Inside the `HousingService` class, paste these functions after the data you just copied.
-        These functions allow dependencies to access the service's data.
+1.  In `src/app/housing.service.ts` and inside the `HousingService` class paste:
+    1.  The variable that you copied from `HomeComponent` in the previous step.
+    1.  The following methods after the data you just copied.
```
Comment (jessicajaniuk) [inline aio/content/tutorial/first-app/first-app-lesson-09.md:76]: `method` vs `function`: What's your preference here, @MarkTechson?

File: aio/content/tutorial/first-app/first-app-lesson-11.md [hunk 2]
```diff
@@ -30,9 +30,8 @@ Route parameters enable you to include dynamic information as a part of your rou
 Perform these steps on the app code in your IDE.
 
 ### Step 1 - Create a new service for your app
-In lesson 10, you added a second route to `src/app/routes.ts`, this route includes a special segment that identifies the route parameter, `id`:
-
-    <code-example format="javascript" language="javascript">
+In lesson 10, you added a secondIn lesson 10, you added a second route to `src/app/routes.ts` which includes a special segment that identifies the route parameter, `id`:
```
Comment (jessicajaniuk) [inline aio/content/tutorial/first-app/first-app-lesson-11.md:33]:
```suggestion
In lesson 10, you added a second route to `src/app/routes.ts` which includes a special segment that identifies the route parameter, `id`:
```

**Review by jessicajaniuk — CHANGES_REQUESTED** [review]
There's a few minor issues here that need addressing, but otherwise looks good.

File: aio/content/tutorial/first-app/first-app-lesson-12.md [hunk 3]
```diff
@@ -99,7 +99,7 @@ This step tests the new form to see that when the form data is submitted to the
 
 1.  In the **Terminal** pane of your IDE, run `ng serve`, if it isn't already running.
 1.  In your browser, open your app at `http://localhost:4200`.
-1.  In your app, right click in the app and from the context menu, choose **Inspect**.
+1.  Right click in the app inside the browser and from the context menu, choose **Inspect**.
```
Comment (MarkTechson) [inline aio/content/tutorial/first-app/first-app-lesson-12.md:102]: "Right click on the app in the browser"

File: aio/content/tutorial/first-app/first-app-lesson-14.md [hunk 4]
```diff
@@ -162,36 +162,40 @@ The data source has been configured, the next step is to update your web app to
...
-The server is now reading data from the `HTTP` request but the components that rely on the service now have errors because they were programmed to use the synchronous version of the service.
+The server is now reading data from the HTTP request but the components that rely on the service now have errors because they were programmed to use the synchronous version of the service.
 
-1.  In `src/app/home/home.component.ts`, update the constructor to use the new asynchronous version of the `getAllHousingLocations` method.
+1.  In `src/app/home/home.component.ts`, update the `constructor` to use the new asynchronous version of the `getAllHousingLocations` method.
 
     <code-example header="" path="first-app-lesson-14/src/app/home/home.component.ts" region="update-home-component-constructor"></code-example>
 
-1.  In `src/app/details/details.component.ts`, update the constructor to use the new asynchronous version of the `getHousingLocationById` method.
+1.  In `src/app/details/details.component.ts`, update the `constructor`` to use the new asynchronous version of the `getHousingLocationById` method.
```
Comment (MarkTechson) [inline aio/content/tutorial/first-app/first-app-lesson-14.md:198]: I think there are too many closing ` characters

File: aio/content/tutorial/first-app/first-app-lesson-14.md [hunk 5]
```diff
@@ -162,36 +162,40 @@ The data source has been configured, the next step is to update your web app to
 
     1.  Update the code to remove `housingLocationList` property and the array containing the data.
 
-    1.  Add a string property called and set the value to `'http://localhost:3000/locations'`
+    1.  Add a string property called `url` and set its value to `http://localhost:3000/locations`
```
Comment (MarkTechson) [inline aio/content/tutorial/first-app/first-app-lesson-14.md:165]: Let's keep the quotes

File: aio/content/tutorial/first-app/first-app-lesson-09.md [hunk 6]
```diff
@@ -90,21 +90,21 @@ In the **Edit** pane of your IDE:
 ### Step 3 - Inject the new service into `HomeComponent`
 
 This step injects the new service into your app's `HomeComponent` so that it can read the app's data from a service.
-In a later lesson, you replace the static data with a live data source to get data as you might in a real app.
+In a later lesson, you will replace the static data with a live data source to get data as you might in a real app.
 
 In the **Edit** pane of your IDE, in `src/app/home/home.component.ts`:
 
-1.  At the top of `src/app/home/home.component.ts`, add the `inject` to the items imported from `@angular/common`. This will import the `inject` function into the `HomeComponent` class.
```
Comment (MarkTechson) [inline aio/content/tutorial/first-app/first-app-lesson-09.md:97]: Let's keep the file name here

File: aio/content/tutorial/first-app/first-app-lesson-09.md (hunk 1 above)
Comment (MarkTechson) [reply aio/content/tutorial/first-app/first-app-lesson-09.md:76]: My care amount is very low here. We can go with method, but then the entire tutorial should follow this convention

File: aio/content/tutorial/first-app/first-app-lesson-09.md [hunk 7]
```diff
@@ -71,14 +71,14 @@ For now, your app's new service uses the data that has, so far, been created loc
 In the **Edit** pane of your IDE:
 
 1.  In `src/app/home/home.component.ts`, from `HomeComponent`, copy the `housingLocationList` variable and its array value.
-1.  In `src/app/housing.service.ts`:
-    1.  Inside the `HousingService` class, paste the variable that you copied from `HomeComponent` in the previous step.
-    1.  Inside the `HousingService` class, paste these functions after the data you just copied.
-        These functions allow dependencies to access the service's data.
+1.  In `src/app/housing.service.ts` and inside the `HousingService` class paste:
```
Comment (MarkTechson) [inline aio/content/tutorial/first-app/first-app-lesson-09.md:74]: Let's prefer the previous format here. You can add methods if you'd like but the repetition here is to help the learners be reinforced on naming and patterns.

**Review by MarkTechson — CHANGES_REQUESTED** [review]

File: aio/content/tutorial/first-app/first-app-lesson-09.md (hunk 1 above)
Comment (bampakoa) [reply aio/content/tutorial/first-app/first-app-lesson-09.md]: @MarkTechson I reverted it back to `functions`. If you have any intention to change that to `methods` later, please let me know and I can scan the whole tutorial to modify it accordingly.

**Review by MarkTechson — APPROVED** [review]
Thanks for fixing up this commit and merging the latest changes.

LGTM

File: aio/content/tutorial/first-app/first-app-lesson-09.md [hunk 8]
```diff
@@ -9,9 +9,9 @@ This tutorial lesson demonstrates how to create an Angular service and use depen
 
 ## What you'll learn
 
-*  Your app has a service to serve the data to your app.
-   At the end of this lesson, the service reads data from local, static data.
-   In a later lesson, you update the service to get data from a web service.
+Your app has a service to serve the data to your app.
+At the end of this lesson, the service reads data from local, static data.
+In a later lesson, you'll update the service to get data from a web service.
```
Comment (bampakoa) [inline aio/content/tutorial/first-app/first-app-lesson-09.md:14]: @MarkTechson added this change to conform to your style for the `will` word.

**Review by jessicajaniuk — APPROVED** [review]
LGTM! Thanks for the fixes!

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit 1ec3a2bc2b26e36f67c4f486725d5e0f4564e821.

**Pull request closed without merging**